filebeat/fileset/fileset.go

```diff
@@ -226,12 +226,12 @@ func resolveVariable(vars map[string]interface{}, value interface{}) (interface{
 }
 
 // applyTemplate applies a Golang text/template. If specialDelims is set to true,
-// the delimiters are set to `{%` and `%}` instead of `{{` and `}}`. These are easier to use
+// the delimiters are set to `{<` and `>}` instead of `{{` and `}}`. These are easier to use
```

Comment:
I wonder if we should have just gone back to {{ ?

Reply:
I am not sure if we can use `{{` and `}}`, because there are two substitutions happening at two different times. See in https://github.com/elastic/beats/pull/5702/files#diff-1b64e79bf3f0b28f0a29965eb2bf0cecR35

Reply:
@ruflin like @kvch said, we will run into the double substitution issue.

Reply:
👍